test/system/src/main/java/io/pravega/test/system/framework/services/docker/BookkeeperDockerService.java

```diff
@@ -65,8 +65,6 @@ private ServiceSpec setServiceSpec() {
         Map<String, String> labels = new HashMap<>();
         labels.put("com.docker.swarm.service.name", serviceName);
 
-        Mount mount1 = Mount.builder().type("volume").source("index-volume").target("/bk/index")
```

Comment:
we were getting  below error,in bookies start by changing `BK_useHostNameAsBookieID` to `true` 

```There are directories without a cookie, and this is neither a new environment, nor is storage expansion enabled. Empty directories are [/bk/journal/current, /bk/ledgers/current, /bk/index/current]```

index-volumes are not persisted in docker-swarm environment.  So removed the index-volume from mount.

Reply:
In Docker Swarm we don't have such a construct like Kubernetes' Persistent Volumes and achieving volume persistence is not straightforward.

Therefore, the workaround we are applying in this PR is preventing the bookie from reusing its existing identity by forcing it to use its IP address as ID instead of the host name.

Also, we don't need to create volumes for any bookie directory because we don't want them to outlive the container.